test/specs/profile/ProfileService.spec.js

```diff
@@ -0,0 +1,251 @@
+    $httpBackend.verifyNoOutstandingRequest();
+  });
+
+  describe('gaTimeFormat', function() {
+
+    it('formats minutes into a human readable text', function() {
+      expect(gaTimeFormat('foo')).to.be('-');
+      expect(gaTimeFormat('123')).to.be('-');
+      expect(gaTimeFormat(-78545)).to.be('-');
+      expect(gaTimeFormat(0)).to.be('0min');
+      expect(gaTimeFormat(49)).to.be('49min');
+      expect(gaTimeFormat(60)).to.be('1h');
+      expect(gaTimeFormat(652)).to.be('10h 52min');
+      expect(gaTimeFormat(123)).to.be('2h 3min');
+    });
+  });
+
+  describe('gaProfile', function() {
+    var profileUrl = 'http://api3.geo.admin.ch/rest/services/profile.json';
+    var goodCoords = [[0, 0], [1, 0], [1, 1]];
+    var goodResult = [
+      {dist: 0.0, alts: {COMB: 429.8}, easting: 592000.0, northing: 221750.0},
+      {dist: 211.5, alts: {COMB: 429.7}, easting: 592177.5, northing: 221865.0}
+    ];
+    var feature = new ol.Feature(new ol.geom.LineString(goodCoords));
+
+    var goodCoords2 = [[0, 0], [10000, 0], [1000, 1000]];
+    var goodResultToFormat = [
+      {dist: 0.0, alts: {}, easting: 592000.0, northing: 221750.0},
+      {dist: 22111.5, alts: {COMB: 1457}, easting: 592177.5, northing: 221865.0}
+    ];
+    var goodResultFormatted = [
+      {dist: 0, alts: {COMB: 0}, easting: 592000, northing: 221750 },
+      {dist: 22.1115, alts: {COMB: 1457}, easting: 592177.5, northing: 221865 }
+    ];
+    var feature2 = new ol.Feature(new ol.geom.LineString(goodCoords2));
+
+    var emptyData = [{alts: {}, dist: 0}];
+    emptyData[0].alts['COMB'] = 0;
+
+    describe('create', function() {
+
+      it('creates an empty chart when no parameters defined', function(done) {
+        var spy = sinon.spy($, 'getScript');
+        gaProfile.create().then(function(profile) {
+          expect(profile).to.be.an(Object);
+          expect(profile.create).to.be.a(Function);
+          expect(profile.update).to.be.a(Function);
+          expect(profile.data).to.eql(emptyData);
+          expect(spy.callCount).to.be(0);
+          expect(profile.elevDiff()).to.be(0);
+          expect(profile.twoElevDiff()).to.eql([0, 0]);
+          expect(profile.slopeDistance()).to.be(0);
+          expect(profile.elPoints()).to.eql([0, 0]);
+          expect(profile.distance()).to.be(0);
+          expect(profile.hikingTime()).to.be(0);
+          done();
+          spy.restore();
+        });
+        $rootScope.$digest();
+      });
+
+      it('creates an chart from a feature', function(done) {
+        $httpBackend.expectPOST(profileUrl).respond(goodResult);
+        gaProfile.create(feature).then(function(profile) {
+          expect(profile).to.be.an(Object);
+          expect(profile.create).to.be.a(Function);
+          expect(profile.update).to.be.a(Function);
+
+          // Test properties
+          expect(profile.data).to.eql(goodResult);
+          expect(profile.elevDiff()).to.be(-0.10000000000002274);
```

Comment:
I prefer to check the exact value. With that we are aware that something has changed in node module.